apps/pom.xml

```diff
@@ -67,6 +68,8 @@
             they are reproducible bit-by-bit no matter when/where they are generated.
             See https://maven.apache.org/guides/mini/guide-reproducible-builds.html -->
         <project.build.outputTimestamp>2022-01-01T00:00:00Z</project.build.outputTimestamp>
+        <bfd.data.fda.version>1.0-SNAPSHOT</bfd.data.fda.version>
+        <bfd.data.npi.version>1.0-SNAPSHOT</bfd.data.npi.version>
```

Comment:
I suppose we will be fixing these versions not matching the format of the rest when we eventually give them a real version.

Reply:
Hopefully we'll replace these with a pipeline soon.  If we ever do another deploy of these I agree we should change them to use the same numbering scheme as the rest of BFD.